docs/release-notes/NuGet-5.9.md

```diff
@@ -0,0 +1,252 @@
+
+* Codespaces:  necessary formatters not registered - [#10467](https://github.com/NuGet/Home/issues/10467)
+
+* Remove net45 as as a target framework from NuGet.Frameworks - [#10470](https://github.com/NuGet/Home/issues/10470)
+
+* Implementation - Add new telemetries to track events related to PMC and Powershell usage. - [#10142](https://github.com/NuGet/Home/issues/10142)
+
+* Only one package shows in the Preview Changes window when there are multiple packages available to update in the Package Manager UI - [#10483](https://github.com/NuGet/Home/issues/10483)
+
+* Empty frameworkReferences groups should be generated when packing multitargeted projects - [#10218](https://github.com/NuGet/Home/issues/10218)
+
+* Hard to see the check-box of package in ‘Updates’ Tab is focused with a dash-line box when navigating through Tab in Blue/Blue (Extra Contrast)/Light themes - [#8963](https://github.com/NuGet/Home/issues/8963)
+
+* Updates Tab checkboxes do not work well with screen-readers - [#10449](https://github.com/NuGet/Home/issues/10449)
+
+* Updating in PMUI causes Object reference not set to an instance of an object - [#9882](https://github.com/NuGet/Home/issues/9882)
+
+* Implementation - Add new telemetries to track events related to PMC and Powershell usage follow up. - [#10478](https://github.com/NuGet/Home/issues/10478)
+
+* Copy-paste error in V2FeedPackageInfo - [#10480](https://github.com/NuGet/Home/issues/10480)
+
+* NuGetPackageFileService fix - use using for disposable memorystream - [#10503](https://github.com/NuGet/Home/issues/10503)
+
+**Nones:**
+
+* Relative links are broken in NuGet.Client debugging guide - [#10100](https://github.com/NuGet/Home/issues/10100)
+
+* Windows SDK 10 link is broken in NuGet.Client Contribution guide - [#10099](https://github.com/NuGet/Home/issues/10099)
+
+* Build of NuGet libraries is broken when assembly signing is disabled  - [#10173](https://github.com/NuGet/Home/issues/10173)
+
+* [Test Failure][Localization] The information from command ‘get-help NuGet’ in PMC is not localized - [#4604](https://github.com/NuGet/Home/issues/4604)
+
+* Design for warning on improper casing for PackageLicenseFile and PackageIcon - [#10049](https://github.com/NuGet/Home/pull/10049)
+
+* No error occurred after packing and installing a package with an invalid .txt format icon - [#9311](https://github.com/NuGet/Home/issues/9311)
+
+* Remove Temp fix on patching - [#8952](https://github.com/NuGet/Home/issues/8952)
+
+* Cannot run Nuget.Client library - [#10390](https://github.com/NuGet/Home/issues/10390)
+
+* Clean-up the contributing docs - [#10399](https://github.com/NuGet/Home/issues/10399)
+
+* Enhance the coding guidelines - add rule for parameter names & namespace names - [#10485](https://github.com/NuGet/Home/issues/10485)
+
+* tfm_raw error happening for some TFMs during pack - [#10656](https://github.com/NuGet/Home/issues/10656)
+
+**StillOpens:**
```

Comment:
I'm not sure we need to have the still opens. We should make sure the release is not specified and then just not include them in the release notes.